src/footer/userList/userList.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+import './userList.scss';
+
+export const users =  [
```

Comment:
this list shouldn't be here. You can save it in separate file, or define this array in Footer.js